### ISSUE TITLE
Import desktop wallet into mobile change from watch only not retained when exit app

### ISSUE THREAD

**Issue by Pleiadien** [ArkEcosystem/mobile-wallet#427]
I had my desktop wallet on my mobile as watch only. I input the passphrase to make it full wallet on mobile which worked fine but when I logged off the mobile app it didn't retain the wallet and it reverted to watch only on next log on.  I have the latest mobile wallet installed on android. 

**Expected Behavior**

**Current Behavior**

**Possible Solution**

**Steps to Reproduce (for bugs)**

**Context**

**Your Environment**

- Version used:
- Environment name and version (e.g. node.js v10.15.3):
- Server type and version:
- Operating System and version:
- Link to your project:

**Comment by ArkEcosystemBot** [comment]
**Thanks for opening this issue!** A maintainer will review this in the next few days and explicitly select labels so you know what's going on.

_If no reviewer appears after a week, a reminder will be sent out._

**Comment by Pleiadien** [comment]
Non of the information I put into the job when I opened it showed up. This needs to be made simpler for non developers to use or no one will use it to report bugs.  

Any I will repeat.....Latest mobile app on android i have a desktop wallet setup as watch only. I imported the passphrase to make it full wallet and that worked but when I logged out of the app the wallet reverted to watch only.  Next log on with pin showed once again watch only.
